src/libYARP_dev/src/devices/AnalogWrapper/AnalogWrapper.cpp

```diff
@@ -140,10 +140,10 @@ AnalogPortEntry &AnalogPortEntry::operator =(const AnalogPortEntry &alt)
 
  // closing anonimous namespace
 
-/**
-  * It reads the data from an analog sensor and sends them on one or more ports.
-  * It creates one rpc port and its related handler for every output port.
-  */
+/*
+ * It reads the data from an analog sensor and sends them on one or more ports.
+ * It creates one rpc port and its related handler for every output port.
+ */
```

Comment:
Sorry for the misunderstanding, I think this one (the one in the cpp, that is not part of the documentation and is is not clear what it is referring to) can be safely removed, and the public documentation in the `.h` can remain as they are.